clarity/src/vm/docs/mod.rs

```diff
@@ -2865,65 +2865,6 @@ mod test {
 
         let contract_id = QualifiedContractIdentifier::local("docs-test").unwrap();
 
-        {
```

Comment:
This code was causing problems with contract existence/non-existence and it seems to be doing the same thing as the "more modern" code below - can anybody confirm that this removal is OK?